client/stylesheets/_modal.scss

```diff
@@ -233,14 +233,55 @@
 	END END END END END END
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+	History Details Modal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+/*~~~~ Button ~~~~*/
 
+/*~~ Button End ~~*/
+
+#profileHistoryFeedback {
```

Comment:
Avoid using id selectors<br>Selector `profileHistoryFeedback` should be written in lowercase with hyphens